components-styled/StyledIcon.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+const StyledIcon = styled.img`
```

Comment:
Ich finde den Namen nicht so gelungen, da du ja deine Icons normalerweise als Component importiert und diese hier vermute ich mal nur für die Icons ist, die du in den Listen anzeigst oder?